daemon/src/projection.rs

```diff
@@ -810,12 +826,21 @@ impl From<Order> for CfdOrder {
                 .settlement_interval
                 .whole_seconds()
                 .try_into()
-                .expect("settlement_time_interval_hours is always positive number"),
+                .context("unable to convert settlement interval")?,
```

Comment:
Previously, we solved this by making the specific field that failed an `Option` rendering it as a `Skeleton` in the UI.

Reply:
I'd say this is not critical for the release and can be done later.